msteams-platform/concepts/extensibility-points.md

```diff
@@ -4,43 +4,47 @@ author: heath-hamilton
 description: Describes where people can discover and use your app in Teams.
 ms.topic: conceptual
 ms.author: lajanuar
-ms.date: 09/22/2020
 ---
-# Entry points for Teams apps
 
-The Teams platform provides a flexible set of entry points where people can discover and use your app. Your app can be as simple as embedding existing web content in a tab or a multi-faceted app that users interact with across several contexts.
+# Entry points for Teams apps
 
-The most successful apps feel native to Teams, so it's important to carefully plan your app's entry points.
+The Teams platform provides a flexible set of entry points, such as team, channel, and chat where people can discover and use your app. Your app can be as simple as embedding existing web content in a tab or a multi-faceted app that users interact with across several contexts.
+The most successful apps are native to Teams, so choose your app's entry points carefully.
 
-## Teams, channels, and chats
+## Collaborate with team, channel, and chat
```

Comment:
```suggestion
## Shared app experiences
```
@KirtiPereira Suggested revision to drive the point home that some apps are used with others, while some are used in individual contexts.